test/spec/validations/osm_api_limits.js

```diff
@@ -2,7 +2,10 @@ describe('iD.validations.osm_api_limits', function () {
     var context;
 
     beforeEach(function() {
-        iD.services.osm = { maxWayNodes: function() { return 10; } };
+        iD.services.osm = {
+            maxWayNodes: () => 10,
+            on: () => undefined,
```

Comment:
this incomplete mock was causing the error:

> `osmService.on` is not a function

in the short time period after the test completes, but before it's cleaned up. only happens on slow machines.